configure

```diff
@@ -49,6 +49,76 @@ while true; do
   # Retry
 done
 
+## Set up MKL related environment settings
+
+while [ "$TF_NEED_MKL" == "" ]; do
+	fromuser=""
+  read -p "Do you wish to build TensorFlow with MKL support? [y/N] " INPUT
```

Comment:
I think you want to make sure that you do not prompt the user until all your PRs are in and things are looking good.
I think its OK to leave in a switch for Intel developers to flip, but that switch should be off by default.